lib/storage/drbd_cmdgen.py

```diff
@@ -345,8 +345,14 @@ def GenLocalInitCmds(self, minor, data_dev, meta_dev, size_mb, params):
 
     return cmds
 
-  def GenNetInitCmd(self, minor, family, lhost, lport, rhost, rport, protocol,
+  def GenNetInitCmds(self, minor, family, lhost, lport, rhost, rport, protocol,
                     dual_pri, hmac, secret, params):
+    cmds = []
+
+    cmds.append(["drbdsetup", "new-resource", self._GetResource(minor)])
```

Comment:
Do these commands also work for 8.3?

Reply:
These commands are inside the `DRBD84CmdGenerator` class, so not relevant for 8.3.